### PR TITLE
PB5 employee project link

### PR DESCRIPTION
**Pull request by amchugh689**
Adds command to setup.sql allowing creation of employee-project table, and addProjectEmpLink.sql allowing customer to link employees and projects.

NB: This requires PB4 to be merged to main as table Project will be added in setup.sql

### CODE REVIEW

**Review by Samuel-BlankAmber — APPROVED** [review]
Stunning work.

**Comment by Samuel-BlankAmber** [comment]
PB4 has been merged so this should work without error

File: sql/addProjectEmpLink.sql
Comment (jkelso662) [inline sql/addProjectEmpLink.sql:6]: CAPS 💔

File: sql/setup.sql
Comment (jkelso662) [inline sql/setup.sql:42]: CAPS 💔

File: sql/setup.sql
Comment (jkelso662) [inline sql/setup.sql:5]: Your tabs are inconsistent with the rest of the file. Not blocking the merge but please fix this in future commits! :)

**Review by jkelso662 — APPROVED** [review]
Please read my comments. CAPS and tabs need sorted but approving so as not to block you. All the rest looks ok to me :)

**Pull request merged**
